### PR TITLE
Hacktoberfest 2020 - coding style for project_euler problem 45 & 16

### PR DESCRIPTION
**Pull request by fa1l**
### **Describe your change:**

Related to #2786

* added link for problem
* added solution function

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by dhruvmanila** [comment]
Please fix one more directory if possible.

**Comment by fa1l** [comment]
Two more files will be enough? There's a race condition between PRs for problems and I don't want to make duplicates or ruin others PRs.

File: project_euler/problem_16/sol1.py [hunk 1]
```diff
@@ -28,7 +30,4 @@ def solution(power):
 
 
 if __name__ == "__main__":
-    power = int(input("Enter the power of 2: ").strip())
-    print("2 ^ ", power, " = ", 2 ** power)
-    result = solution(power)
-    print("Sum of the digits is: ", result)
+    print(f"Sum of the digits is {solution(1000)}")
```
Comment (dhruvmanila) [inline project_euler/problem_16/sol1.py:33]: Why are you changing the original solution? Please undo the changes

File: project_euler/problem_16/sol2.py [hunk 2]
```diff
@@ -25,4 +27,4 @@ def solution(power):
 
 
 if __name__ == "__main__":
-    print(solution(int(str(input()).strip())))
+    print(f"Sum of the digits is {solution(1000)}")
```
Comment (dhruvmanila) [inline project_euler/problem_16/sol2.py:30]: Same as above

Comment (fa1l) [reply project_euler/problem_16/sol2.py]: my bad, thought that without input() it will be better. Like in some other solutions in problems.

**Review by dhruvmanila — APPROVED** [review]
